layouts/_default/landing.html

```diff
@@ -0,0 +1,192 @@
+{{ define "body-class" }}
+  text-[16px] bg-beige font-serif leading-normal text-s-9
+{{ end }}
+{{ define "html-class" -}}
+  scroll-smooth
+{{- end }}
+{{ define "static-head" }}
+  {{ partialCached "header/static-tw.html" . }}
+{{ end }}
+
+{{ define "header" }}
+  <header data-ga-category="header">
+    {{ block "nav" . }}
+      {{ partialCached "tw/nav.html" . }}
+    {{ end }}
+    {{ block "banner" . }}
+      {{ if .Param "banner-active" }}
+        {{ partialCached "tw/banner.html" . }}
+      {{ end }}
+    {{ end }}
+    {{ block "topper" . }}
+      {{ if .Param "topper-active" }}
+        {{ partialCached "tw/promo-topper.html" . }}
+      {{ end }}
+    {{ end }}
+    {{ block "promo" . }}
+      {{ partialCached "tw/promo-header.html" . }}
+    {{ end }}
+  </header>
+{{ end }}
+
+{{ define "main" }}
+  <article class="bg-white px-4 pt-12 pb-2 md:pb-6 overflow-visible">
+    <header>
+      <div class="mx-auto w-full max-w-screen-md">
+        <h1
+          class="font-sans text-2xl sm:text-4xl md:text-5xl font-black leading-none tracking-wide"
+        >
+          {{- .Title -}}
+        </h1>
+      </div>
+      <div class="mx-auto w-full max-w-prose">
+
+        {{ partial "tw/image-hero.html" . }}
+
+      </div>
+    </header>
+    {{ with .Content }}
+      <div class="mx-auto w-full max-w-screen-md">
+        <div class="mt-6" data-ga-category="content">
+          <div
+            x-data
+            x-intersect.once="$report({ target: $el })"
+            data-ga-action="view:content-start"
```

Comment:
This triggers an event in Google Analytics when someone gets to the start of the page. I guess it should probably just be pulled out into a partial since we use it in a couple of different places. Maybe the partial could just take the action name as an argument, and be used like `{{ partial "view-trigger.html" "content-start" }}`. I'm not sure what folder it should go in though. Might need to make layout/partials/js or something.

Reply:
I've added `js/view-trigger.html`

https://github.com/spotlightpa/poor-richard/commit/0d74333090faf20930fe54bac4fe54357a0993eb